docs/versioned-plugins/codecs/s3plain-v2.0.5.asciidoc

```diff
@@ -1,4 +1,4 @@
-:plugin: s3_plain
+:plugin: s3plain
```

Comment:
I might have led you astray in a previous review comment. The plugin name here is wrong because the actual name in the repo is s3plain. We'll need to change this in the repo because it's misleading. The underscore character was resulting in broken links.